pkgs/tools/misc/libspeedhack/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, multiStdenv, fetchFromGitHub }:
+
+multiStdenv.mkDerivation rec {
+  pname = "libspeedhack";
+  version = "0.1-x86-multilib";
+
+  src = fetchFromGitHub {
+    owner = "evg-zhabotinsky";
+    repo = pname;
+    rev = version;
+    hash = "sha256-+ymV3hWeNtnaUBPkbELhloQ1UVf1vAwXnPOMj2aun54=";
+  };
+
+  patches = [
+    ./fix_paths.patch
+  ];
```

Comment:
seems like the patch does a bit more then just fixing the paths and it also doesn't seem to nixos specific so would be nice if we could get it upstream